js/src/api/contract/contract.js

```diff
@@ -14,8 +14,14 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-import Abi from '../../abi';
+import BigNumber from 'bignumber.js';
+import { intersection } from 'lodash';
 
+import Abi from '~/abi';
+import WalletAbi from '~/contracts/abi/wallet.json';
+import WalletsUtils from '~/util/wallets';
```

Comment:
As discussed, these two lines above are UI and implementation specific, and doesn't belong in the API. The API layer takes what is provided and encodes that, sends it, decodes the responses, the role is not to make decisions and tweaks on behalf of the caller (caller is either us or other developers).

Changes made here is valid for the app, just needs a different home outside of `~/api`